modules/client-configuration/pages/supported-features.adoc

```diff
@@ -41,22 +41,22 @@ The operating system you run on a client is supported by the organization that s
 | Salt Clients
 
 | {sle} 15
-| {x86_64}, {ppc}, {ibmz}, ARM
+| {x86_64}, {ppc64le}, {ibmz}, ARM
 | {check}
 | {check}
 
 | {sle} 12
-| {x86_64}, {ppc}, {ibmz}, ARM
+| {x86_64}, {ppc64le}, {ibmz}, ARM
 | {check}
 | {check}
 
 | {sle} 11
-| {x86}, {x86_64}, {ipf}, IBM {ppc}, {ibmz}
+| {x86}, {x86_64}, {ipf}, IBM {ppc64le}, {ibmz}
```

Comment:
SLE11 is ppc64 (not ppc64le). You can verify for your self what tools are supported for each client by looking in SCC or the sub-projects of https://build.opensuse.org/project/subprojects/systemsmanagement:Uyuni:Stable

```suggestion
| {x86}, {x86_64}, {ipf}, {ppc64}, {ibmz}
```